src/pages/view/SupersetDashboard.js

```diff
@@ -36,13 +36,26 @@ const reducer = (state, action) => {
     }
 }
 
+class SupersetGatewayUnavailableError extends Error {
+    constructor() {
+        super(
+            i18n.t(
+                'Could not load dashboard content because Superset Gateway is unavailable'
```

Comment:
I think there was a suggestion for a different error message in the feature channel.